### DISCUSSION TITLE
Added well formatted content for gitignore and gitattributes

### DISCUSSION THREAD

**Discussion by hindermath** [holgerflick/flixaccounting#18] [Ideas]
I think it is good idea recommendation to insert well formatted content for the two files gitignore and gitattributes. I use tis in my projects where I use this files. The content I use is creatable from
https://gitignore.io/ and https://gitattributes.io/. I configure them with the keywords DELPHI, FreePascal, Pascal and/or Lazarus.
Here are the exact URLs for getting the correct formatted content of this two files for every project you are using this files:
https://[gitattributes.io/api/delphi%2Cpascal](https://gitattributes.io/api/delphi%2Cpascal)
https://www.[toptal.com/developers/gitignore/api/delphi,freepascal,lazarus](https://www.toptal.com/developers/gitignore/api/delphi,freepascal,lazarus)

**Comment by holgerflick** [comment]
These are excellent comments. I hope if I close this people will be able to read this comment in the future. First time I am a moderator in a GitHub discussion. So if it goes wrong, we have to reopen or write again ;-)

**Comment by holgerflick** [comment]
Please provide replacement files as pull request if you wish.
